### ISSUE TITLE
Deprecate explicit bulking in favor of auto-bulk

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#9247]
AFAICS you can only make additional errors when using explicit bulking.

**Comment by namdre** [comment]
current auto-bulk code only checks  origin,time and vClass. It may be necessary to also store vehicle maximum speed.

I recall that we tried always-auto-bulking in the past and then disabled it again for some reason.

**Comment by behrisch** [comment]
Yes, I can think of reasons why auto-bulking fails. The question is rather do we have use cases where we want to manually enforce bulking although the auto bulker would not do it?

**Comment by namdre** [comment]
> The question is rather do we have use cases where we want to manually enforce bulking although the auto bulker would not do it?

I think we don't have those.

**Comment by namdre** [comment]
We should add an option that defines a tolerance for auto-bulking. In the first step we could set the default to no-auto-bulk and use it to test the bulking correctness in a test suite variant.
If everything works, we can switch the default to auto-bulk with some level of tolerance (i.e. if speedFactor is capped at 2 and maxSpeed >= maxRoadSpeed * 2 we can safely auto-bulk vehicles)
